nuxeo-compound-documents/pom.xml

```
@@ -29,18 +29,17 @@
     </dependency>
     <dependency>
       <groupId>org.nuxeo.ecm.platform</groupId>
-      <artifactId>nuxeo-platform-imaging-core</artifactId>
-      <type>test-jar</type>
+      <artifactId>nuxeo-platform-video</artifactId>
```

In order to keep historic:
```suggestion
      <artifactId>nuxeo-platform-imaging-test</artifactId>
```